sphynx/lynxkite-sphynx/networkit_compute_attribute.go

```diff
@@ -34,7 +34,6 @@ func init() {
 				Defined: make([]bool, len(vs.MappingToUnordered)),
 			}
 			var result networkit.DoubleVector
-			defer networkit.DeleteDoubleVector(result)
```

Comment:
So wait, then we don't need to delete the "right thing", either?

Reply:
> So wait, then we don't need to delete the "right thing", either?

This tries to delete `nil`. If I move this line after the `switch`, it crashes with `free(): invalid pointer`. I took that to mean we don't need to free it. But I have no idea. 😓